deep_qa/models/reading_comprehension/bidirectional_attention.py

```diff
@@ -222,10 +225,18 @@ def _set_padding_lengths_from_model(self):
         # self.num_sentence_words.
         self._set_text_lengths_from_model_input(self.model.get_input_shape_at(0)[1][1:])
 
-    def _get_instance_sorting_keys(self) -> List[str]:  # pylint: disable=no-self-use
+    @overrides
+    def get_instance_sorting_keys(self) -> List[str]:  # pylint: disable=no-self-use
         return ['num_passage_words', 'num_question_words']
 
+    @overrides
+    def get_padding_memory_scaling(self, padding_lengths: Dict[str, int]) -> int:
```

Comment:
Is dynamic padding only working with Bidaf at the moment? We should probably try and implement it for the other models reasonably soon because it's quite a big feature to have only for a single model. That's definitely a separate issue though.